dev-lang/rust/rust-crossdev.patch

```diff
@@ -0,0 +1,52 @@
+diff --git a/dev-lang/rust/rust-1.44.1.ebuild b/dev-lang/rust/rust-1.44.1.ebuild
```

Comment:
I am wondering why this patch `rust-crossdev.path` exists.
Maybe you wanted to keep a separate crossdev patch, which can be easily applied in the upcoming releases?

I understand the motivation.
But I am not sure if that is a good idea.
Almost for every rust release we needed to adjust the crossdev patch, simply because of tiny changes from upstream Gentoo ebuilds.
Now with this separate file, we need to update the crossdev patch twice, the first in the actual ebuild, and the second in `rust-crossdev.patch`. Not to mention that the latter has a complicated syntax.

Reply:
A commit diff is easily lost, that's why I thought a file would be good. But yes, we don't have it normally and it also has drawbacks, I'll remove it again.